### PR TITLE
Initial Pull Request

### PR DESCRIPTION
**Pull request by bx44**
This is the main development branch of the project, this is an initial pull request for a complete version of it.
This request contains the index.html file, the CSS files and image resources.

### CODE REVIEW

File: css/style.css
Comment (pelzolga123) [inline css/style.css]: It's better to delete commented code.

**Review by pelzolga123 — CHANGES_REQUESTED** [review]
Project requires changes.

**Comment by pelzolga123** [comment]
## Hi Barak!  
#### Your project looks :+1: greate!

Here what you need to change, before this project can be approved:
#### General
* [x] Project should references [Bootstrap CSS file](https://getbootstrap.com/docs/4.3/getting-started/introduction/#css) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/using_bootstrap/bootstrap_requirement.png) from [Odin](https://www.theodinproject.com/courses/html5-and-css3/lessons/using-bootstrap)) Right now you just added this files to your project, that's not the right way.
* [x] Pull Request should have meaningful title and a short description ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_prs.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))

#### Navbar
* [x] It should contain Sign in and Subscribe links ([link to page](https://www.newsweek.com/))

#### Main content
* [x] This section is missing:
image
* [x] This part needs a little correction to look more like an original:
image

Original:
image


#### Responsiveness
* [x] Tablet version is messed a little:
image

* [x] In mobile version navbar is messed:
image

### Please kindly fix the above issues and send another code review request.

If you have any questions or need help, you can message me on slack (@ Olga Pelts). *Happy coding*

**Review by mosesogwo — CHANGES_REQUESTED** [review]
### Hi @bx44 , good job. But we need to make some little changes for approval:
1. Sign in and Subscribe in the header should be links.
2. Align this section of the page properly:
> **Yours:** 
image
> **Original:** 
image

**Kindly make these changes and submit another code review request**

### Status: Changes requested.

**Review by mosesogwo — APPROVED** [review]
### :trophy: Congratulations! Your project is approved.

You may now merge to master.

### Status: Approved!

**Pull request merged**
